ci/dockerd-entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
Please add a comment explaining how this shell script is intended to be used

Reply:
This file is copied from codebuild repo, better keep it as is.